src/OpenLoco/src/GameCommands/GameCommands.cpp

```diff
@@ -419,9 +420,9 @@ namespace OpenLoco::GameCommands
         }
 
         // advanced errors
-        if (_9C68D0 != (void*)-1)
+        if (_9C68D0 != (const TileElement*)-1)
```

Comment:
TileManager.h defines `TileElement* const kInvalidTile = reinterpret_cast<TileElement*>(static_cast<intptr_t>(-1))`. Perhaps we can generalise that somewhere?